servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -263,20 +186,14 @@ def _metadata_response(id, job):
         extensions=extensions.get_extensions(job))
 
 
-def _provider_type():
-    return current_app.config['PROVIDER_TYPE']
+def _auth_token():
+    auth_header = request.headers.get('Authentication')
+    if auth_header:
+        components = auth_header.split(' ')
+        if len(components) == 2 and components[0] == 'Bearer':
+            return components[1]
+    return None
 
 
-def _query_jobs_result(job, project_id=None):
-    return QueryJobsResult(
-        id=job_ids.dsub_to_api(project_id, job['job-id'], job.get('task-id')),
-        name=job['job-name'],
-        status=job_statuses.dsub_to_api(job),
-        # The LocalJobProvider returns create-time with milliescond granularity.
-        # For consistency with the GoogleJobProvider, truncate to second
-        # granularity.
-        submission=job['create-time'].replace(microsecond=0),
-        start=job.get('start-time'),
-        end=job['end-time'],
-        labels=labels.dsub_to_api(job),
-        extensions=extensions.get_extensions(job))
+def _provider_type():
```

Comment:
nit: I think CH had another comment suggesting to move this into `utils/providers.py`, seems like that would be the best place for this (might have to remove the _ if its used outside of providers.py for anything). 

Could also potentially move `_auth_token()` there since I think we only use the token to authenticate dsub, could also have a new `utils/auth.py`